main/EventSystem.cpp

```diff
@@ -2442,27 +2440,28 @@ void CEventSystem::ExportDomoticzDataToLua(lua_State *lua_state, uint64_t device
 			_hardwareNames[ID] = tlist;
 		}
 	}
+	
+	Json::Value tempjson;
+	m_webservers.GetJSonDevices(tempjson, "true", "", "dzvID", "", "", "", true, false, false, 0, "");
 
-	//_log.Log(LOG_STATUS, "%d devices in table.", m_devicestates.size());
-
+	if (tempjson["result"].size() != m_devicestates.size())
+	{
+		_log.Log(LOG_ERROR, "Json and devicestates size mismatch!");
+	}
+	
 	lua_createtable(lua_state, 0, 0);
 
 	// First export all the devices.
 	typedef std::map<uint64_t, _tDeviceStatus>::iterator it_type;
+	int ii2 = 0;
```

Comment:
after ii, there is jj, kk, ll, mm
But sometimes you can label the variable better then ii,jj,...